authors/apps/authentication/tests/test_authentication.py

```diff
@@ -0,0 +1,87 @@
+from django.urls import reverse
+from django.contrib.auth import get_user_model
+from rest_framework import status
+from rest_framework .test import APITestCase, APIClient
+
+
+class UserManagerTestCase(APITestCase):
+    """Usermanager TestCase."""
+
+    def setUp(self):
+        self.client = APIClient()
+
+        self.user_registration = {
+            "user": {
+                "username": "mwinel",
+                "email": "mwinel@live.com",
+                "password": "12345678"
+            }
+        }
+
+        self.user_login = {
+            "user": {
+                "email": "mwinel@live.com",
+                "password": "12345678"
+            }
+        }
+
+    def test_create_user(self):
+        """Test create user."""
```

Comment:
@mwinel This is testing for many things in one test. You should separate them so you follow TDD principles. `A Test should only test for one thing and should be independent.` So you can have 10 assertions under one test.